drivers/volume/portworx/portworx.go

```diff
@@ -647,6 +647,10 @@ func (d *portworx) RecoverDriver(n node.Node) error {
 			Cause: err.Error(),
 		}
 	}
+	// TODO: refactor the function into two pieces, entermaintenance and exitmaintenance. So we don't need to have
```

Comment:
I like the idea of creating 2 new functions and calling them from here. Then, we can call the new functions directly from our tests. If it is not too much work, we should just refactor now. Otherwise, maybe we could add a parameter for the sleep interval?